modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15182,6 +15187,7 @@ select-the-publication-to-move-the-selected-changes-to=Select the publication to
 select-the-publication-x-to-review-the-change=Select the publication {0} to review the change.
 select-the-ratings-type-for-the-following-applications=Select the ratings type for the following applications.
 select-the-reassignment-type=Select the reassignment type.
+select-the-root-object-path=Select the Root Object Path
```

Comment:
@danvb112 Maybe BChan will consider this as a sentence, but let's wait to see :)